build/travis/configure-travis-ci-xsite.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+export MAKE_DATADIR_WRITABLE=true
+export GO111MODULE=on
+export INITCONTAINER_IMAGE=quay.io/quay/busybox:latest
+export TESTING_NAMESPACE=${TESTING_NAMESPACE-namespace-for-testing}
+export KUBECONFIG=~/kind-kube-config.yaml
+
+if ! [ -x "$(command -v kind)" ]; then
+  curl -Lo kind https://github.com/kubernetes-sigs/kind/releases/download/"${KIND_VERSION}"/kind-linux-amd64 && chmod +x kind && sudo mv kind /usr/local/bin/
+fi
+if ! [ -x "$(command -v kubectl)" ]; then
+  curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+fi
+# This requires for local (outside Travis CI) cleanup and run
```

Comment:
can we run this script locally to test?

Reply:
that is cool! :)
would it make sense to put those instructions in some readme file?

Reply:
Added notes to the main README.md file